src/cli/flags.go

```diff
@@ -41,6 +41,13 @@ func Flags(command FlagsType) []cli.Flag {
 				Required: true,
 				Category: "deploy",
 			},
+			&cli.StringFlag{
+				Name:     "app-port",
+				EnvVars:  []string{"KKA_APP_PORT"},
+				Required: false,
+				Category: "deploy",
+				Value:    "80",
```

Comment:
I wouldn't define a default value but if the port is not specified we should fallback to knative behaviour, meaning we should not add the "ports:" section to the yaml